h/static/scripts/dropdown-menu-controller.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
Could we possibly start putting non-entrypoint JavaScript files in a directory of their own? Having one big flat folder full of JavaScript makes it very difficult to understand how the code is structured.